### DISCUSSION TITLE
FastAPI integration with Grapql - Starlette GraphQLApp is deprecated

### DISCUSSION THREAD

**Discussion by chandrasekharsimhala** [fastapi/fastapi#3052] [Questions] [answered]
Hi All,

I have seen this statement in the Starlette github rep, "GraphQL support in Starlette is **deprecated** as of version 0.15 and will
be removed in a future release. Please consider using a third-party library
to provide GraphQL support." 

I'm worried as I have built API's on FASTAPI using Starlette GraphQLApp. What is the future plan on FastAPI integration with Graphql. Is this integration is already being worked on or is there any other way I can integrate Graphql with FASTAPI?

Can someone throw some light on this.

**Comment by psgandalf** [comment]
I bump this up as it is important for me aswell.

/Micke

**Comment by tiangolo** [comment, marked as answer]
You can read the new docs here: https://fastapi.tiangolo.com/advanced/graphql/

In short, for a new project, use Strawberry, and to preserve old code based on Graphene, you can use the libraries listed there.

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
